classes/services/PKPFileService.inc.php

```diff
@@ -85,9 +85,25 @@ public function add($from, $to) {
 		if (is_resource($stream)) {
 			fclose($stream);
 		}
+		$mimetype = $this->fs->getMimetype($to);
+
+		// Check and override ambiguous mime types based on file extension
+		$pathinfo = pathinfo($to);
```

Comment:
This is totally fine but just FYI you can also use `pathinfo($to, PATHINFO_EXTENSION)` to return just the extension: https://www.php.net/manual/en/function.pathinfo.php

Reply:
Thanks! I've fixed it, I updated PR anyway